plugin.json

```diff
@@ -96,10 +96,17 @@
       },
       {
         "key": "DisplaySubscriptionNameInNotifications",
-        "display_name": "Display subscription name in notifications",
+        "display_name": "Display subscription name in notifications:",
         "type": "bool",
         "help_text": "Display subscription name in post when a subscription posts to a channel",
         "default": false
+      },
+      {
+        "key": "EnableWebhookEventLogging",
+        "display_name": "Enable Webhook Event Logging:",
+        "type": "bool",
+        "help_text": "Allow the plugin to log the event from webhook.",
```

Comment:
Can we also state that the log level needs to be set to `DEBUG` in order to see these webhook events in the logs?